pennylane_lightning/src/simulators/lightning_kokkos/gates/GateFunctorsGenerator.hpp

```diff
@@ -204,19 +203,19 @@ template <class PrecisionT, bool adj = false> struct generatorIsingZZFunctor {
 
     Kokkos::View<Kokkos::complex<PrecisionT> *> arr;
 
-    std::size_t rev_wire0;
-    std::size_t rev_wire1;
-    std::size_t rev_wire0_shift;
-    std::size_t rev_wire1_shift;
-    std::size_t rev_wire_min;
-    std::size_t rev_wire_max;
-    std::size_t parity_low;
-    std::size_t parity_high;
-    std::size_t parity_middle;
+    size_t rev_wire0;
+    size_t rev_wire1;
+    size_t rev_wire0_shift;
+    size_t rev_wire1_shift;
+    size_t rev_wire_min;
+    size_t rev_wire_max;
+    size_t parity_low;
+    size_t parity_high;
+    size_t parity_middle;
```

Comment:
Note that changes like this aren't idiomatic C++ and may break with future compiler versions (implementation detail specific). I am fine with this for now, but if it starts complaining with CT later or GCC 13.1 we can re-add all the `std::` calls.

We require `cstddef` to be included somewhere for `std::size_t`, which is a C++ header, and `stddef.h` for `size_t` which is a C header.

If you want to keep the `size_t` only, you can likely add a `using std::size_t` equivalent somewhere in the anon namespace, while including `cstddef`

Reply:
Thanks for pointing this out. I have read something similar on StackOverflow, but also that in practice most compilers equate the two. I did the change since I realized that LQubit was mostly using `size_t` directly unlike LKokkos. I could have done the opposite, but there would have been more changes. If needed, we could now just add `using std::size_t` in all files using `size_t` as you pointed out.

Reply:
To get this bug fix out asap, can we just pretend I didn't poke the bear and revert the `size_t`-changes?